ImperatorToCK3/Source/Imperator/Provinces/Pops.cpp

```diff
@@ -0,0 +1,43 @@
+#include "Pops.h"
+#include "Pop.h"
+#include "Log.h"
+#include "ParserHelpers.h"
+
+
+void ImperatorWorld::Pops::loadPops(std::istream& theStream)
+{
+	registerKeys();
+	parseStream(theStream);
+	clearRegisteredKeywords();
+}
+
+void ImperatorWorld::Pops::registerKeys()
+{
+	registerRegex("\\d+", [this](const std::string& thePopID, std::istream& theStream) {
+		const auto popStr = commonItems::singleItem(thePopID, theStream);
+		if (popStr.find('{') != std::string::npos)
```

Comment:
Do you expect you might have [popid] = none again? Jesus, PDX.

Reply:
I'm surprised you're still surprised 😃:

> 51272={
> 	type="citizen"
> 	culture="pasargadi"
> 	religion="zoroaster"
> }
> 51273={
> 	type="citizen"
> 	culture="pasargadi"
> 	religion="zoroaster"
> }
> 51274=none